run_snafu.py

```diff
@@ -128,9 +129,11 @@ def main():
 def process_generator(index_args, parser):
     benchmark_wrapper_object_generator = generate_wrapper_object(index_args, parser)
 
+    drop_cache()
     for wrapper_object in benchmark_wrapper_object_generator:
         for data_object in wrapper_object.run():
             for action, index in data_object.emit_actions():
+                drop_cache()  # before the next
```

Comment:
This will drop cache at the beginning of snafu. Shouldn't be dropped for each sample. The same as ceph cache dropping: https://github.com/cloud-bulldozer/snafu/blob/master/fio_wrapper/fio_wrapper.py#L73-L74

Reply:
@rsevilla87 thanks for taking a look.   I disagree - emit_actions() is what actually runs a sample and yields objects for elasticsearch.   So by placing drop_cache() after the emit_actions call we are ensuring that cache is dropped before the *next* sample, and we call drop_cache() before the first sample is collected as well.  What I am concerned about is that emit_actions() yields > 1 document per sample, so this could result in excessive cache dropping.   I couldn't see a good way to get drop_cache() to run exactly once per sample.   @acalhounRH ?